src/assets/i18n/cs.json

```diff
@@ -1551,6 +1551,7 @@
       "pricing_definition_delete_error": "Error occurred while deleting a pricing definition, check the logs",
       "pricing_definition_error": "Error occurred while retrieving the pricing definition, check the logs",
       "pricing_definition_time_range_error": "Please select different start and end time",
+      "pricing_min_max_error": "Please select a maximum value greater than the minimum.",
```

Comment:
No '.' at the end.

Reply:
No 'Please select..' but  'Maximum price must be greater than the minimum price'.
Change also the rest of the messages.
Thanks!